contrib/pyln-proto/pyln/proto/zbase32.py

```diff
@@ -23,34 +24,76 @@
 ]
 
 
-def bitarray_to_u5(barr):
-    assert len(barr) % 5 == 0
+def _message_to_bitarray(message: bytes) -> bitstring.ConstBitStream:
+    """Encodes a message as a bitarray with length multiple of 5."""
+    barr = bitstring.ConstBitStream(message)
+    padding_len = len(barr) % 5
```

Comment:
Technically padding is `5 - (len(barr) % 5)`, but that's splitting hairs here :-)

Reply:
Actually yeah, it used to be called `rem` for remainder by after all the name changing now it's rather not representing its meaning anymore. Will update to fit the description.